src/main/java/software/sigma/internship/entity/Fund.java

```diff
@@ -0,0 +1,45 @@
+package software.sigma.internship.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity(name = "fund")
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+public class Fund {
+
+    @Id
+    @Column(name = "id",
+            nullable = false)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
Please avoid using primitive wrappers where it's unnecessary (generated ids and not null fields)

Reply:
So, it's better:
1. to use primitive long for ids instead of a wrapper Long
2. not to use _nullable_ parameter in Column annotation, when there is the automatic generation

Did I get your message?

Reply:
1) Use long for ids and int for StatisticalData fields, because they will be always present
2) Yep